lib/commands/addon.js

```diff
@@ -15,6 +15,13 @@ module.exports = NewCommand.extend({
     { name: 'skip-git', type: Boolean, default: false, aliases: ['sg'] },
     { name: 'yarn', type: Boolean }, // no default means use yarn if the blueprint has a yarn.lock
     { name: 'directory', type: String, aliases: ['dir'] },
+    {
+      name: 'lang',
+      type: String,
+      default: '',
```

Comment:
I think technically we do not have a default. 

```suggestion
```

As it stands this says the default is `''` (empty string) but the blueprint only adds `lang=` if a value is set...

Reply:
> I think technically we do not have a default.
> 
> As it stands this says the default is `''` (empty string) but the blueprint only adds `lang=` if a value is set...

As in, there's no consensus about whether or not to have a valid language code by default or what that code would be? AFAIK there isn't, and I definitely think that discussion exceeds the scope of this 'tooling-only' RFC.

I absolutely agree with you that we should use `undefined` instead. My rationale for choosing the empty String initially was that it was falsy enough to handle the 'no-input' logic planned for the utility *and* it would serve as a convenient vehicle for receiving a default language code *in the event that such a decision ever got made*. However, the choice to use the String could definitely be (mis)interpreted as a biased one, i.e., that it treats the outcome of any discussion about a default as a foregone conclusion. I'd love to rule that out, and `undefined` should do that.

I think I've covered the change appropriately with [this set of changes](https://github.com/josephdsumner/ember-cli/pull/6/files). The only notable change in control is the utility: it asserts that the input is defined before trying to do any formatting / evaluation / validation.